news/2024/2024-06-24-4-digit-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,167 @@
+> 
+> Our first step was to tell the oversleeper to set more alarms and make sure they could hear phone calls. You can never be too sure that the bed isn't extra comfy in the morning. In the following weeks, we made sure to practice together as a team to ensure more confident rosters on every map. Otherwise, we moved on. The match didn't really have an effect on us mentally and we knew that we could win with better preparation.
+
+::{#blanktap}::
+
+[![](/wiki/shared/news/2024-06-24-4-digit-world-cup-2024-concludes/blanktap-banner.png)](https://osu.ppy.sh/users/10137131)
+
+> Hello, I'm BlankTap. I'm a bit of a tournament player and recently more known as a commentator, probably one that gets a bit too excited at times. I've been commentating actively for one and half years now and had the chance to contribute regarding 4WC's casting.
+
+**BlankTap** has been making a name for himself in recent years thanks to his high-energy commentary, and he was an integral part of the casting team during this year's 4WC.
+
+---
+
+*What were your thoughts participating in your first world-cup-style tournament as a commentator? How did you prepare yourself?*
+
+> I was very excited and basically in disbelief. The overturning of what's initially a rejection of my commentating application energized me up to prove that this will not be a decision to be regretted upon. I really do want to say thank you and that I am eternally indebted to those who believed in me that made this possible.
+> 
+> Regarding preparation, I don't think I can explain much other than that I basically did nearly nothing. I've only discovered the archetypes of commentating recently and while this wouldn't fly when you're an analyst or a color caster, play by play casting needs much less intricate knowledge of a team's strategy or skillset. It revolves more around how the current map will go, where it helps that I do play parts of the mappools for fun and have some knowledge regarding how a map generally would go for from the experience I've accumulated as a tournament player myself.
+> 
+> In short, my preparation consists of filling up my water bottle, sitting comfortably in my chair, and going hard when the moment needs it to be.
+
+*This year's 4WC had a lot of upsets in the earlier rounds. Do you believe that this is something that could happen more frequently from now on and does it make commentating more exciting for you?*
+
+> Regarding the plentiful early round upsets that were witnessed in this iteration, it really has to do with how the earlier rounds consist of lower star rating maps and the big focus on a player's consistency rather than their skillcap. I do think that every country is as good as each other up until a certain star rating. There are no safe rounds in a world cup where every nation sends in their best of the best. High seeding is nothing more but an intimidation factor if you don't back it up.
```

Comment:
```suggestion
> Regarding the plentiful early round upsets that were witnessed in this iteration, it really has to do with how the earlier rounds consist of lower star rating maps and the big focus on a player's consistency rather than their skill cap. I do think that every country is as good as each other up until a certain star rating. There are no safe rounds in a world cup where every nation sends in their best of the best. High seeding is nothing more but an intimidation factor if you don't back it up.
```